doc/reference/src/main/asciidoc/executing.asciidoc

```diff
@@ -86,14 +86,21 @@ Since Arquillian in-container tests are executed in a remote JVM, the results of
 
 === Running the Tests In the Container - SE part
 
-To execute full TCK testsuite you have to run tests within SE group as well. For this you need to execute "weld-se" Maven profile from the JBoss TCK runner POM file as follows: 
+To execute full TCK testsuite you have to run tests within SE group as well. 
+SE tests make use of link:$$https://github.com/arquillian/arquillian-container-se$$[Arquillian container SE].
+This way the tests are executed in a separate JVM instance with isolated and configurable classpath.
+The Arquillian container does not start CDI container in any way - this is still done directly in the tests.
```

Comment:
"directly in the tests **using CDI SE bootstrap API and `javax.enterprise.inject.se.SeContainerInitializer `**"?